src/components/IceBreaker.svelte

```diff
@@ -0,0 +1,68 @@
+<script>
+  import clsx from 'clsx';
+  import { onMount } from 'svelte';
+  import { _ } from 'svelte-i18n';
+  import { board, password } from '../store.js';
+  import { decrypt, encrypt, checkBoardPassword } from '../encryption.js';
+
+  import Textarea from './Textarea.svelte';
+  import EncryptedText from './EncryptedText.svelte';
+
+  let showIceBreaking = false;
+  let iceBreakingEditMode = false;
+  let newIceBreakingText = '';
+
+  let classes = '';
+  let className = '';
+  export { className as class };
+
+  async function checkIceBreak() {
+    newIceBreakingText = $board.ice_breaking || '';
+    showIceBreaking = newIceBreakingText !== '';
+    return showIceBreaking;
+  }
+
+  async function startEdit() {
+    if ($board.owner && (await checkBoardPassword($board, $password))) {
+      iceBreakingEditMode = true;
+      newIceBreakingText = await decrypt($board.ice_breaking, $password);
+    }
+  }
+
+  function cancelEdit() {
+    iceBreakingEditMode = false;
+  }
+
+  async function submitEdit() {
+    $board.ice_breaking = await encrypt(newIceBreakingText, $password);
+    iceBreakingEditMode = false;
+  }
+
+  onMount(async () => {
+    checkIceBreak();
+  });
+
+  $: classes = clsx(className, 'p-3', 'mx-auto');
+</script>
+
+{#if showIceBreaking}
+  <div class={classes}>
+    <div class="card text-center" on:click={startEdit}>
+      <div class="card-body">
+        <h5 class="card-title">{$_('splash.icebreaking')}</h5>
+        {#if iceBreakingEditMode}
+          <Textarea
```

Comment:
You can add the 'data-name' attribute here directly, then the cypress test will be able to find it.

Reply:
All right. This way it works (surrounding with a div):

```
<div data-name="ice-breaker-message">
  <EncryptedText bind:text={$board.ice_breaking} />
</div>
```

Putting it in the EncryptedText element doesn't work.